<launch>
    <node pkg="jinho" type="jinho_node" name="jinho_node" output="screen"/>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find jinho)/rviz/config.rviz"/>
</launch>